index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="main.css">
+</head>
+<body>
+  <nav>
+    <ul>
+      <li><a href="#">Portfolio</a></li>
+      <li><a href="#">About</a></li>
+      <li><a href="#">Contact</a></li>
+    </ul>
+  </nav>
+
+  <main>
+    <header>
+      <h1>Hey there. I'm Edozie. <br> <span id="title-span">I'm a software developer</span></h1>
+      <p id="text">
+        I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <div class="socials">
+        <div><a href="#"><img src="/pics/github.png" alt="github logo"></a></div>
+        <div><a href="#"><img src="/pics/linkedin.png" alt="linkedin logo"></a></div>
+        <div><a href="#"><img src="/pics/Enabled.png" alt="enabled"></a></div>
+        <div><a href="#"><img src="/pics/twitter.png" alt="twitter"></a></div>
+        <div><a href="#"><img src="/pics/medium.png" alt="medium"></a></div>
+      </div>
```

Comment:
- [ ] The social media icons should be wrapped in a `ul` tag, and each image icon should be  in an li tag wrapped in an `a`  tag, please fix this 👍🏿